src/Symfony/Bundle/FrameworkBundle/Logger/Logger.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Logger;
+
+use Psr\Log\AbstractLogger;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LogLevel;
+
+/**
+ * Minimalist PSR-3 logger designed to write in stdout, stderr or any other stream.
+ *
+ * @internal
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+final class Logger extends AbstractLogger
+{
+    private static $defaultOutputByLevel = array(
+        LogLevel::EMERGENCY => 'php://stderr',
+        LogLevel::ALERT => 'php://stderr',
+        LogLevel::CRITICAL => 'php://stderr',
+        LogLevel::ERROR => 'php://stderr',
+        LogLevel::WARNING => 'php://stderr',
+        LogLevel::NOTICE => false,
+        LogLevel::INFO => false,
+        LogLevel::DEBUG => false,
```

Comment:
Instead of using using by-level output streams I would defined an unique stream and a minimum logged level, I don't think logging different levels to separate streams is a common enough use case for a minimal logger.

Reply:
I prefer to keep this feature, for instance it allows to redirect some levels on `stdout` and some others on `strerr`, or to create a file on disk per level.
This level to stream mapping and the formatter one allows to have a very flexible solution in just 100 LOC (100 LOC is what we can call minimal). For all more advanced use cases, Monolog must be installed and that's is.

Reply:
redirecting logs to stdout is not a good practice and should not be encouraged, so that's not a valid use case IMHO. Creating a file is also not a use case. I tend to agree with @jvasseur, let's keep it minimal. It's not "Monolog should be installed for advanced use cases", it's more like "Monolog must be installed for any configurable logger". If not, drawing the line won't be easy.

Reply:
I was looking for feature parity with the `log` package of the Go language (that is minimalist). This package has the ability to change the output (https://golang.org/pkg/log/#SetOutput) and to set the format. That's all.

Reply:
Go only allows one single output, not one per level, isn't it?

Reply:
Yes

Reply:
Makes sense to configure the output globally, not by level indeed.